src/modules/hype_chat/styles.module.css

```diff
@@ -0,0 +1,13 @@
+.hideHypeChatMessages {
+  div[class$='paid-pinned-chat-message-content-wrapper'] {
+    display: none !important;
+  }
+}
+
+.hideHypeChatButton {
+  button:has(
```

Comment:
can we scope this down further so we're not filtering every button on the page? alternatively, we could leverage the dom watcher to check this within a scope. this would then work on firefox too..